templates/activity-stream/defaults.schema.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mozPipelineMetadata": {
+    "expiration_policy": {
+      "delete_after_days": 180
+    }
+  }
```

Comment:
Namespace-level defaults were listed as a future extension in the proposal, but as I was implementing this, it really seems necessary sooner rather than later, so I've included here support for files called `defaults.schema.json` within each namespace directory. Like `README`s, these files are not copied into `schemas/`.

Reply:
If I'm reading this correctly, then I'm in agreement that we can defer proper namespace-level metadata to a future improvement.